.env.example

```diff
@@ -0,0 +1 @@
+CKEDITOR_PRODUCTIVITY_LICENSE="Your productivity licence"
```

Comment:
Note that the license should be retrieved from the GitHub secrets at build time and so not visible in the source code

Reply:
Is .env (or .env.example), dotenv needed here? from my tests locally it's also possible for webpack to just pick this up from an environment variable, which we only need to set when running the github action workflow. It might simplify code a bit if this works with github actions.

Also, do we need to add this part in the github actions workflow for this PR? We can add the trial license we have for now in our github secrets to test the whole flow.

Reply:
@AKarmanov I've given you admin access for the time needed to make this work